src/UnrollLoops.cpp

```diff
@@ -11,6 +11,18 @@ using std::pair;
 namespace Halide {
 namespace Internal {
 
+struct RemoveLikelies : public IRMutator {
```

Comment:
Should we perhaps move this to the internal section of IROperator? I feel like I've needed to copy-paste it once already.

Reply:
done